nixos/tests/home-assistant-config.nix

```diff
@@ -0,0 +1,65 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+let
+  configDir = "/var/lib/hass";
+  configFile = "${configDir}/configuration.yaml";
+  port = 4000;
+
+  mkHass = { machineAttrs ? {}, hassAttrs ? {} }:
+      { pkgs, ... }:
+      {
+        services.home-assistant = {
+          inherit configDir port;
+          enable = true;
+        } // hassAttrs;
+      } // machineAttrs;
+in {
+  name = "home-assistant-config";
+
+  nodes = {
+    hassMinimal = mkHass {};
+
+    hassTimeZone = mkHass {
+      machineAttrs = { time.timeZone = "UTC"; };
+    };
+
+    hassNoDefault = mkHass {
+      hassAttrs = { applyDefaultConfig = false; };
+    };
```

Comment:
I don't think we are going to need all those tests. They don't test much but still needs to be maintained and they are expensive to evaluate. It would be fine if they only would generate the configuration but they spawn actual VMs.

Reply:
Hey @Mic92 thanks for the feedback. This was my feeling as well, but I wasn't sure how to do unit tests in Nix. Perhaps I just missed some examples or documentation. Is there anything you could point me to that might help?